doc/whats_new.rst

```diff
@@ -192,6 +194,10 @@ Changelog
 
     - Add :func:`mne.report.Report.add_slider_to_section` methods to :class:`mne.report.Report` by `Teon Brooks`_
 
+    - Extended functionality of :func:`mne.Epochs` selection by ``event_id``: allow selecting multiple conditions based on tags separated by ``/``s, by `Jona Sassenhagen`_
+
+    - Add :func:`mne.statistics.linear_regression_raw` function for the estimation of potentially overlapping linear effects of potentially continuous predictors in continuous data ("rER[P/F]s"), by `Jona Sassenhagen`_
+
```

Comment:
then don't put it in the next release notes but below and double check it's
not already there...